appveyor.yml

```diff
@@ -0,0 +1,11 @@
+os: Visual Studio 2017
```

Comment:
I did [40+ builds](https://ci.appveyor.com/project/ctaggart/visualfsharp/history) on AppVeyor before submitting this pull request. It is useful for those of use who would like to contribute. We can remove it before this is ready to merge.